pkg/controller/kieapp/kieapp_controller.go

```diff
@@ -149,6 +154,10 @@ func (reconciler *Reconciler) Reconcile(request reconcile.Request) (reconcile.Re
 		if err != nil {
 			return reconcile.Result{}, err
 		}
+		if resourceType == reflect.TypeOf(consolev1.ConsoleLink{}) && len(delta.Added) == 1 {
+			instance.Status.ConsoleLink = delta.Added[0].GetName()
```

Comment:
This looks really ugly! Trying to understand why we need it!

Reply:
I needed a way to set the generated name to the consoleLink field in the status. If I try the KieApp UID that shouldn't be necessary.
I don't like it either, too many magic values but I have been trying to find a better way without success.